tests/test_parser_error.py

```diff
@@ -1,11 +1,14 @@
 from __future__ import annotations
+
```

Comment:
Is this an unrelated change?

Reply:
Good catch! That seems to have been added by some refactoring tool semi-automatically. I removed it!